ntbs-service/DataAccess/DataQualityRepository.cs

```diff
@@ -69,17 +72,26 @@ public async Task<IList<Notification>> GetNotificationsEligibleForDataQualityClu
 
         public async Task<IList<Notification>> GetNotificationsEligibleForDataQualityTreatmentOutcome12Alerts()
         {
-            return await GetNotificationsEligibleForTreatmentOutcomeAlertByAgeInMonths(12);
+            return await GetNotificationQueryableForNotifiedDataQualityAlerts()
+                .Where(n => (n.ClinicalDetails.TreatmentStartDate ?? n.NotificationDate) < DateTime.Now.AddYears(-1))
+                .Where(n => _treatmentOutcomeService.IsTreatmentOutcomeNeededAtXYears(n, 1))
```

Comment:
I know we discussed at length the merits of trying to make this stuff performant - and decided it wasn't worth going too far down the rabbit hole - do you know if the first `where` clause is in db or in memory?

Reply:
In memory sadly